exchange/exchange.go

```diff
@@ -63,6 +63,9 @@ type exchange struct {
 type seatResponseExtra struct {
 	ResponseTimeMillis int
 	Errors             []openrtb_ext.ExtBidderError
+	// httpCalls is the list of debugging info. It should only be populated if the request.test == 1.
+	// This will become response.ext.debug.httpcalls.{bidder} on the final Response.
+	httpCalls []*openrtb_ext.ExtHttpCall
```

Comment:
This member begins with a lower-case letter indicating that's private. The other members are public. I'm curious as to why this one is treated differently.

Reply:
I made it private same as we already have it in `pbsOrtbSeatBid struct`. But here, yes, it should be public. Good catch! Fixed.